R/doctor.R

```diff
@@ -91,7 +93,21 @@ dr_github <- function(path = ".") {
 
   if (identical(Sys.getenv("GITHUB_PAT"), ""))
     msg[["PAT"]] <- paste("* GITHUB_PAT environment variable not set",
-      "(this is not critical unless you want to install private repos)")
+      "(this is not necessary unless you want to install private repos",
+      "or connect local repos to GitHub)")
+
+  pkg <- as.package(path)
+  desc_path <- file.path(pkg$path, "DESCRIPTION")
+  desc <- read_dcf(desc_path)
+  desc <- compact(desc[c("URL", "BugReports")])
```

Comment:
I think it's more important to warn that Url and BugReports fields are empty. 

Do you think that people mindlessly copying-and-pasting `<USERNAME>/<REPO>` is a common failure mode?